RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -2100,6 +2101,81 @@ boolean acquireGumItem(item it)
 	return (have + 1) == item_amount(it);
 }
 
+boolean acquireTotem()
+{
+	//this function checks if you have a valid totem for casting turtle tamer buffs with. Returning true if you do. If you don't, it will attempt to acquire one in a reasonable manner.
+
+	//first check if there is a valid totem in inventory or equipped, return true if there is.
+	
+	if ((0 < item_amount($item[primitive alien totem])) || (have_equipped($item[primitive alien totem])))
+	{
+		return true;
+	}
+	if ((0 < item_amount($item[Flail of the Seven Aspects])) || (have_equipped($item[Flail of the Seven Aspects])))
+	{
+		return true;
+	}
+	if ((0 < item_amount($item[Chelonian Morningstar])) || (have_equipped($item[Chelonian Morningstar])))
+	{
+		return true;
+	}
+	if ((0 < item_amount($item[Mace of the Tortoise])) || (have_equipped($item[Mace of the Tortoise])))
+	{
+		return true;
+	}
+	if ((0 < item_amount($item[Ouija Board, Ouija Board])) || (have_equipped($item[Ouija Board, Ouija Board])))
+	{
+		return true;
+	}
+	if ((0 < item_amount($item[turtle totem])) || (have_equipped($item[turtle totem])))
+	{
+		return true;
+	}
```

Comment:
Should use a plural in a foreach here so we're not repeating ourselves unnecessarily and use possessEquipment e.g.
```
foreach totem in $items[turtle totem, Mace of the Tortoise, Chelonian Morningstar, Flail of the Seven Aspects, Ouija Board, Ouija Board, primitive alien totem]
{
	if (possessEquipment(totem))
	{
		return true;
	}
}
```

Reply:
Fair enough, but I see a potential issue. $items seperates each item using a comma, but "Ouija Board, Ouija Board" is an item that has a comma as part of its name.

How about something like the following? where the first function tests all but Ouija Board, Ouija Board and the second one only tests Ouija Board, Ouija Board?
```
foreach totem in $items[turtle totem, Mace of the Tortoise, Chelonian Morningstar, Flail of the Seven Aspects, primitive alien totem]
	{
		if (possessEquipment(totem))
		{
			return true;
		}
	}
if ((0 < item_amount($item[Ouija Board, Ouija Board])) || (have_equipped($item[Ouija Board, Ouija Board])))
	{
		return true;
	}
```

Reply:
use the escape code for the comma e.g. `Ouija Board\, Ouija Board`

Reply:
change made.